Source/scrollrt.cpp

```diff
@@ -1206,9 +1206,15 @@ void DrawView(const Surface &out, Point startPosition)
 		// force redrawing or debug stuff stays on panel on 640x480 resolution
 		force_redraw = 255;
 		char debugGridTextBuffer[10];
+		bool megaTiles = IsDebugGridInMegatiles();

```

Comment:
It's starting to introducing a few debug features in this file that aren't left out for release builds, so it would be good if we could clean things up a bit in the near future.